src/Network/Network/help/New-AzRouteServer.md

```diff
@@ -14,8 +14,8 @@ Creates an Azure RouteServer.
 
 ```
 New-AzRouteServer -ResourceGroupName <String> -RouteServerName <String> -HostedSubnet <String>
- -Location <String> [-Tag <Hashtable>] [-Force] [-AsJob] [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
- [-Confirm] [<CommonParameters>]
+ -PublicIpAddress <PSPublicIpAddress> -Location <String> [-Tag <Hashtable>] [-Force] [-AsJob]
```

Comment:
Please update help file as PublicIpAddress is optional now.

Reply:
updated